build/commands/lib/util.js

```diff
@@ -481,11 +481,16 @@ const util = {
   // So, we should copy our pre-installed files to overwrite upstream pre-installed files.
   // After checking, pre-installed files are copied to gen dir and they are used to compile.
   // So, this copying in every build doesn't affect compile performance.
-  updateOmahaMidlFiles: () => {
-    console.log('update omaha midl files...')
+  updateMidlFiles: () => {
+    console.log('update midl files...')
     const srcDir = path.join(config.braveCoreDir, 'win_build_output', 'midl', 'google_update')
     const dstDir = path.join(config.srcDir, 'third_party', 'win_build_output', 'midl', 'google_update')
     fs.copySync(srcDir, dstDir)
+
```

Comment:
I would replace this duplicate code by a loop over `["google_update", "brave"]`.

Reply:
done